ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-669] Add dependants section

## Description of change
Display dependants in apply

## Link to relevant ticket
[CRIMAPP-669](https://dsdmoj.atlassian.net/browse/CRIMAPP-669)
[Figma designs](https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=8959-130668&mode=design&t=Sje6bHZGhRsUJl8J-0)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:
<img width="711" alt="Screenshot 2024-03-28 at 14 24 32" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/3effb2c2-802a-436d-87b8-cdd4b90a1176">
<img width="733" alt="Screenshot 2024-03-28 at 14 04 16" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/7848c2eb-08f6-4495-9334-68c13f761bb6">


## How to manually test the feature

Start an application and fill in dependants information 
Confirm dependants section is shown in the review application screen and completed application screen and appears like the first screenshot
Confirm that the change links route you to the correct page

Start a section application and select no to having dependants
Confirm dependants section is shown in the review application screen and completed application screen and appears like the second screenshot
Confirm that the change link routes you to the correct page

[CRIMAPP-669]: https://dsdmoj.atlassian.net/browse/CRIMAPP-669?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
